### ISSUE TITLE
Add information for local ip when using VPN on main screen and install certificates instructions

### ISSUE THREAD

**Issue by almaz010891** [ProxymanApp/Proxyman#1570]
## Description
When I use VPN, Proxyman app shows me only remote IP-adress. 
<img width="565" alt="image" src="https://user-images.githubusercontent.com/4091750/223166168-3a2074d2-0cfe-4cb9-bb34-68901712bf75.png">

<img width="481" alt="image" src="https://user-images.githubusercontent.com/4091750/223167701-c8f1dc28-5aa2-4319-984d-fc337fb0d794.png">

But for capturing traffic from mobile devices it's better to use local IP-adress. Please add information for local P-adress when using VPN on main screen and install certificates instructions

## Why this feature/change is important?
Because for mobile testing is important to know your local IP-adress

**Comment by NghiaTranUIT** [comment]
Thanks. I'm able to reproduce it when using the VPN. 

It should always display the local Address, instead of the IP from the VPN 👍

**Comment by NghiaTranUIT** [comment]
@almaz010891 Fixed with this Beta build: https://download.proxyman.io/beta/Proxyman_4.3.1_Fix_VPN_IP.dmg

Proxyman will ignore the IP from the VPN and fetch the correct one 👍

**Comment by almaz010891** [comment]
Hi! Still reproduce in v 4.4.0

**Comment by NghiaTranUIT** [comment]
What VPN you're using @almaz010891?

**Comment by almaz010891** [comment]
Cisco Anyconnect (Corporate VPN)

**Comment by almaz010891** [comment]
Still reproduce in Version 4.12.0 (49300)

**Comment by Agrailag** [comment]
@NghiaTranUIT hello, i see its Done, but i also have problem, that i saw my vpn adress instead proxyman while using Cisco Any Connect. Will be appriciate for fixing this one.
Proxyman Version 5.8.0 (50800) | macOS: Version 12.7.6 (Build 21H1320)

**Comment by NghiaTranUIT** [comment]
@Agrailag can you show me what is the out of this command: `ipconfig getifaddr en0` ?

Is it the same with the IP what displays on Proxyman ?

**Comment by Agrailag** [comment]
> Is it the same with the IP what displays on Proxyman ?

No, its not. 192.168.88.204 in terminal using ipconfig getifaddr en0. And vpn ip-adress in Proxyman.

**Comment by vladyslavsosiuk** [comment]
Hi @NghiaTranUIT 
I can still reproduce this bug in Proxyman 6.2.0

This looks like a regression related to a change in the VPN environment — either a change in our company’s VPN configuration or in the VPN client itself. Poxyman used to display the correct IP address before, so this feels like an edge case that Proxyman might not be handling properly.

`ipconfig getifaddr en0` returns the local IP address, but Proxyman shows the VPN IP address instead.

**Comment by NghiaTranUIT** [comment]
@vladyslavsosiuk can you try this Beta build: https://assets.proxyman.com/beta/Proxyman_6.2.0_Try_fixing_IPv4.dmg

I found a better solution and might fix your problem 👍

**Comment by vladyslavsosiuk** [comment]
> @vladyslavsosiuk can you try this Beta build: https://assets.proxyman.com/beta/Proxyman_6.2.0_Try_fixing_IPv4.dmg
> 
> I found a better solution and might fix your problem 👍 

Yes, it works now! Thank you @NghiaTranUIT

**Comment by NghiaTranUIT** [comment]
Awesome, I will include this fix in the next release 👍
